schemas/constructs/v1alpha3/relationship.json

```diff
@@ -28,11 +21,7 @@
     "kind": {
       "$ref": "../core.json#/definitions/inputString",
       "description": "Kind of the Relationship. Learn more about relationships - https://docs.meshery.io/concepts/logical/relationships.",
-      "enum": [
-        "Hierarchical",
-        "Edge",
-        "Sibling"
-      ]
+      "enum": ["Hierarchical", "Edge", "Sibling"]
```

Comment:
@MUzairS15, please confirm that lowercased versions of these enums are supported, too.

Reply:
That would mean intercepting the validation, so let's keep them lowercase. Clients can format it when needed to display, ok?

Reply:
Sounds great. I will rely upon you to ensure that our current clients are compatible with lowercase values. Please suggest a change here and let’s merge and release.

Reply:
Relationship policies are currently compatible?

Reply:
yes